README.md

```
@@ -1,109 +1,161 @@
-## <img src="./public/img/icon.svg" style="width: 2.5rem; vertical-align: middle;"/> API Monitor - Chrome Developer Tools extension
+## <img src="./public/img/icon.svg" style="width: 2.5rem; vertical-align: middle;"/> API Monitor DevTools - Chrome Developer Tools extension - API panel
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)